### PR TITLE
[Citations] Endpoint to fetch all citations associated with user org(s)

### PR DESCRIPTION
**Pull request by yattias**
- New endpoint: `saved_org_citations`
- Fix logic bug that prevented papers without a PDF from being saved in ref manager
- Helper function `get_user_organizations`
- Adding reviews to search serializer

### CODE REVIEW

File: src/citation/views/citation_entry_view.py [hunk 1]
```diff
@@ -357,6 +368,19 @@ def upload_library(self, request):
...
+        user = User.objects.get(id=request.user.id)
+        organizations = get_user_organizations(user)
+        saved_citations = self.get_queryset().filter(
+            organization_id__in=[o.id for o in organizations]
+        )
+
+        serializer = MinimalCitationEntrySerializer(saved_citations, many=True)
+        return Response(serializer.data)
+
+    @action(detail=False, methods=["get"], permission_classes=[AllowAny])
```
Comment (koutst) [inline src/citation/views/citation_entry_view.py:383]: Did the permission_classes on user_citations change?

**Review by koutst — APPROVED** [review]
LGTM

File: src/citation/views/citation_entry_view.py (hunk 1 above)
Comment (yattias) [reply src/citation/views/citation_entry_view.py:383]: Good catch, I was testing and forgot to revert back

File: src/citation/tests/test_view.py [hunk 2]
```diff
@@ -13,6 +14,49 @@ def setUp(self):
...
+            source=self.authenticated_user.organization,
+            user=self.organization_user,
+        )
+        self.client.force_authenticate(
+            self.organization_user, organization=self.authenticated_user.organization
+        )
+
+        # Next, we need to create a project
+        response = self.client.post(
+            f"/api/citation_project/",
```
Comment (gzurowski) [inline src/citation/tests/test_view.py:33]: Nit: String formatting not needed here.

**Review by gzurowski — APPROVED** [review]
LGTM 👍

**Pull request merged**